crates/starknet-server/src/api/models/mod.rs

```diff
@@ -41,6 +41,23 @@ pub enum BlockId {
     Tag(Tag),
 }
 
+impl From<BlockId> for starknet_rs_core::types::BlockId {
```

Comment:
let's import` starknet_rs_core::types::`. If there're name collisions you can use `as`